pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -139,13 +140,13 @@ func GetEnvironment(cr *api.KieApp, service kubernetes.PlatformService) (api.Env
 }
 
 func setProductLabels(cr *api.KieApp, env *api.Environment) {
-	setObjectLabels(cr, &env.Console)
-	setObjectLabels(cr, &env.Dashbuilder)
+	setObjectLabels(cr, &env.Console, getFormattedComponentName(cr, "businesscentral"))
```

Comment:
does not hardcode values here, businesscentral is different than decision central, this value should come from env.Console.

same for the hardcoded values below.